R/qualification-comparison-time-profile.R

```diff
@@ -87,8 +87,8 @@ addOutputToComparisonTimeProfile <- function(outputMapping, simulationDuration,
     project = outputMapping$Project,
     simulation = outputMapping$Simulation
   )
-  simulation <- loadSimulation(simulationFile)
-  simulationResults <- importResultsFromCSV(simulation, simulationResultsFile)
+  simulation <- ospsuite::loadSimulation(simulationFile)

```

Comment:
@pchelle This would be a candidate for `loadFromCache=true`, right?

Reply:
I have added issue #583 for that because most (if not all) of the tasks are currently simply loading the simulation without any `loadFromCache=true`